src/index.js

```diff
@@ -84,6 +85,8 @@ activitiesRunningInstances.forEach((value, key) => {
   }
 });
 
+
+// add CSS classes to waiting edge instances
```

Comment:
the 2 parts are not consistent
- activities: overlays then CSS classes
- edges: CSS classes then overlays

Can we do it in the same order?

Reply:
yes you're right